extensions/wikia/ContentReview/ContentReviewImportJS.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Wikia\ContentReview;
+
+class ImportJS {
+	const IMPORT_SCRIPTS = 'ImportJS';
```

Comment:
I'd consider renaming this const, since what it ultimately is is a title of a page. :)